Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3.8.5-slim-buster
+
+RUN apt update && apt install -y \
+    unzip \
+    git \
+    cloc \
```

```suggestion
    cloc \
    curl \
```
We need curl to install poetry.